Dockerfile

```diff
@@ -1,5 +1,6 @@
 #############      builder       #############
-FROM eu.gcr.io/gardener-project/3rd/golang:1.15.5 AS builder
+ARG GCR_PULL_URL=eu.gcr.io/gardener-project/3rd/
+FROM ${GCR_PULL_URL}golang:1.15.5 AS builder
```

Comment:
Can you explain why this is required?

Reply:
https://github.com/gardener/gardener/pull/3544#discussion_r575993840

Reply:
So `golang:1.15.8` is multi-arch and `eu.gcr.io/gardener-project/3rd/golang:1.15.8` is not? Why we don't fix our images instead of introducing similar workarounds?

Reply:
@ccwienk has spent a lot of effort to improve the CI/CD and we also have dockerhub account, so we should not see the rate limits and can try to switch back to the official `golang:<version>` image.

Reply:
@vpnachev happy to do that. Can you verify this?

Reply:
> Can you verify this?

I have migrated one pipeline back to dockerhub  and quickly after hit the rate limits :cry:

Reply:
I added a card about it https://github.com/orgs/gardener/projects/6#card-55193026.